scripts/lint-style.lean

```diff
@@ -83,10 +129,14 @@ def checkInitImports : IO Bool := do
     return true
   return false
 
+/-- Verify that every file in the `scripts` directory is documented in `scripts/README.md`. -/
```

Comment:
nit: I'd write this docstring without a period, as this is not a complete sentence (imperative, not indicative)
This may be a matter of style --- I won't insist on this.